File: 3Week/14Day/java/Algo06/src/test/ten.java
package test;

public class ten {
	public String[] solution(String my_string) {
		String[] str = my_string.split(" ");
		int count = 0;
		
		for(String s : str) {
			if(!s.equals("")) {
				count++;
			}
		}
        String[] answer = new String[count];
        count = 0;
        for(String s : str) {
			if(!s.equals("")) {
				answer[count++] = s;
			}
		}
        
        return answer;
    }
}

File: 4Week/16Day/과제/ten.py
import re

def solution(my_string):
    list_string = list(re.sub(r'[^ 0-9]', '', my_string))
    answer = []
    for i in range(len(list_string)):
        answer.append(int(list_string[i]))
    answer.sort()
    return answer